### DISCUSSION TITLE
possible to define a function?

### DISCUSSION THREAD

**Discussion by danielb2** [casey/just#1197] [Q&A] [answered]
Makefiles allow to define a function 

```makefile
define show
   tput setaf 3; echo $(1); tput setaf 8
endef

stop:
   @for i in ${SERVICES}; do $(call show, stop $$i); docker stop $$i; done
```

which I can then call. is this possible in just ?

**Comment by casey** [comment, marked as answer]
There was a [PR](https://github.com/casey/just/pull/1069) starting to implement custom functions, but it looks like it's stalled. So not possible at the moment, unfortunately.

**Reply by danielb2** [reply]
Thank you 😌
